homeassistant/components/mqtt/light/schema_json.py

```diff
@@ -157,6 +157,8 @@ async def async_setup_entity_json(
 class MqttLightJson(MqttEntity, LightEntity, RestoreEntity):
     """Representation of a MQTT JSON light."""
 
+    _attributes_extra_blocked = MQTT_LIGHT_ATTRIBUTES_BLOCKED
```

Comment:
I suggest we make this a frozenset since it will be used as a constant and should not be mutable. This is extra important since we'll set it as a class attribute.
```suggestion
    _attributes_extra_blocked = frozenset(MQTT_LIGHT_ATTRIBUTES_BLOCKED)
```